crates/bevy_pbr/src/atmosphere/functions.wgsl

```diff
@@ -379,3 +400,142 @@ fn zenith_azimuth_to_ray_dir(zenith: f32, azimuth: f32) -> vec3<f32> {
     let cos_azimuth = cos(azimuth);
     return vec3(sin_azimuth * sin_zenith, mu, -cos_azimuth * sin_zenith);
 }
+
+fn ray_sphere_intersect(r: f32, mu: f32, sphere_radius: f32) -> vec2<f32> {
```

Comment:
this isnt atmosphere-specific, could be in bevy_render utils or something like that.

Reply:
id put it wherever `orthonormalize` lives

Reply:
Yes good idea, i thought of this too! I will push a commit to address your feedback